en/docs/setup/analytics/configure-analytics-for-elk-stack.md

```diff
@@ -2,29 +2,51 @@
 
 ## Step 1 - Setup APK
 
-1. Start by following the instructions outlined in [Customize Configurations](../Customize-Configurations.md). These instructions will guide you through the process of acquiring the `values.yaml` file, which you will then use to tailor the analytics configurations to your specific needs. Open `values.yaml` file.
+1. Start by following the instructions outlined in [Customize Configurations](../Customize-Configurations.md). These instructions will guide you through the process of acquiring the `values.yaml` file.
+2. Open the `values.yaml` file, and add the above configuration to the gatewayRuntime section under dp.
```

Comment:
add the ~above~ below configuration

Reply:
Fixed in https://github.com/wso2/docs-apk/pull/584

Sentence changed to: Open the `values.yaml` file and set the following configuration under the `wso2.apk.dp.gatewayRuntime` section.